reconcile/terraform_resources.py

```diff
@@ -107,21 +108,28 @@ def setup(print_only, thread_pool_size):
                      QONTRACT_TF_PREFIX,
                      thread_pool_size,
                      oc_map)
-    working_dirs = ts.dump(print_only)
+    working_dirs, error = ts.dump(print_only)
+    if error:
+        cleanup_and_exit(status=error, working_dirs=working_dirs)
     tf = Terraform(QONTRACT_INTEGRATION,
                    QONTRACT_INTEGRATION_VERSION,
                    QONTRACT_TF_PREFIX,
                    working_dirs,
                    thread_pool_size)
     existing_secrets = tf.get_terraform_output_secrets()
     ts.populate_resources(tf_query, existing_secrets)
-    ts.dump(print_only, existing_dirs=working_dirs)
+    _, error = ts.dump(print_only, existing_dirs=working_dirs)
+    if error:
+        cleanup_and_exit(status=error, working_dirs=working_dirs)
 
     return ri, oc_map, tf
 
 
-def cleanup_and_exit(tf=None, status=False):
-    if tf is not None:
+def cleanup_and_exit(tf=None, status=False, working_dirs={}):
+    if tf is None:
```

Comment:
Better if we log the `status` here if its not set to `False`?

Reply:
I missed that its a bool, might be able to change it so that we can log the error message. But non-blocking on this PR